data-structures.md

```diff
@@ -79,18 +104,19 @@ Blocks are currently serialized simply by CBOR marshaling them, using lower-came
 
 ```go
 type Message struct {
+  
 	To   Address
 	From Address
-	
+
 	// When receiving a message from a user account the nonce in
 	// the message must match the expected nonce in the from actor.
 	// This prevents replay attacks.
 	Nonce Integer
 
 	Value Integer
-    
-    GasPrice Integer
-    GasLimit Integer
+
+	GasPrice Integer
```

Comment:
@whyrusleeping, @dignifiedquire: is there a canonical description for `Integer`? In other words, what exactly do we mean by integer here? Is this a 32 byte binary quantity? Perhaps it's a bigInt in form LEB128?

Reply:
Do we know the size of these Integers?